platform/src/main/java/org/hillview/dataset/api/IJson.java

```diff
@@ -28,9 +28,10 @@
 import java.lang.reflect.Type;
 import java.time.Instant;
 
-// Unfortunately this module introduces many circular dependencies, because it has
-// to register various type adaptors.
-
+/**
```

Comment:
This is not a proper javadoc comment. a Javadoc comment should describe what the described object is doing, and not what the shortcomings are. This should stay as a regular comment.